addons/auth_signup/data/ir_cron_data.xml

```diff
@@ -4,10 +4,10 @@
         <field name="name">Users: Notify About Unregistered Users</field>
         <field name="model_id" ref="model_res_users"/>
         <field name="state">code</field>
-        <field name="code">model.send_unregistered_user_reminder()</field>
+        <field name="code">action = model.send_unregistered_user_reminder(batch_size=100)</field>
```

Comment:
@Julien00859 and I have discussed the issue of returning the value with this `action = ...`.  That's way too bizarre and unintuitive.  We eventually agreed that making cron methods return `(done, remaining)` was not really clear and the need for `action = ...` make it quite confusing.